apps/originals-explorer/server/routes.ts

```diff
@@ -728,6 +850,32 @@ export async function registerRoutes(app: Express): Promise<Server> {
     }
   });
 
+  // Serve DID log at .well-known path (did.jsonl)
+  // According to DID:WebVH spec, the log should be available at:
+  // /.well-known/did/{path-segments}/did.jsonl
```

Comment:
@cursoragent we don't use .well-known if we are using a path. read the spec again